xwf/gateway/integ_tests/gw/entrypoint.sh

```diff
@@ -63,5 +63,9 @@ dnsmasq
 echo "run DHCP server"
 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid gw0 &
 
+echo "set mms"
+iptables -t mangle -A FORWARD -o gw0 -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1400
```

Comment:
Can you put long options names too?

Reply:
I don't understand

Reply:
As a general rule of thumb, use long flags everywhere as it increases readability https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation